includes/elements/container.php

```diff
@@ -1009,7 +1009,7 @@ protected function register_border_controls() {
 				'fields_options' => [
```

Comment:
I am still seeing physical properties.
e.g. 
--border-top-left-radius

Reply:
These CSS variables are not implemented in the container SCSS. 
I've also checked `main`.

Have looks like these variables have been orphaned since they where created and replaced with 
`--border-radius`

Was `--border-top-left-radius` (right|top|bottom)  a fallback meant to be removed later?
Or have we dropped functionality on another Merged PR by mistake?

Reply:
Can you please delete these variables inside container.php and inside nested-accordion.php?